packages/orbit-components/src/Toast/ToastMessage.tsx

```diff
@@ -122,15 +79,26 @@ const ToastMessage = ({
           setPaused(false);
         }}
       >
+        <div
+          className="rounded-large bg-white-normal absolute left-0 top-0 z-[10] h-full w-full animate-[toast-light_var(--toast-message-duration)_linear] opacity-10 will-change-transform"
```

Comment:
before z-index was 1, not 10, don't think it will change something, but let's keep it as it was :) 

```suggestion
          className="rounded-large bg-white-normal absolute left-0 top-0 z-[1] h-full w-full animate-[toast-light_var(--toast-message-duration)_linear] opacity-10 will-change-transform"
```

Reply:
adjusted